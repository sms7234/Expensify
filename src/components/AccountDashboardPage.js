import React from 'react';
import AccountList from './AccountList';
import AccountSummary from './AccountSummary';
import AccountListFilters from './AccountListFilters';

const AccountDashboardPage = () => (
  <div>
    <AccountSummary />
    <AccountListFilters />
    <AccountList />
  </div>
)

export default AccountDashboardPage;
