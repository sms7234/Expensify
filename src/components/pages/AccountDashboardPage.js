import React from 'react';
import AccountList from '../lists/AccountList';
import AccountSummary from '../independents/AccountSummary';
import AccountListFilters from '../filters/AccountListFilters';

const AccountDashboardPage = () => (
  <div>
    <AccountSummary />
    <AccountListFilters />
    <AccountList />
  </div>
)

export default AccountDashboardPage;
