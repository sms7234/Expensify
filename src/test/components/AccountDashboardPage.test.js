import React from 'react';
import {shallow} from 'enzyme';
import AccountDashboardPage from '../../components/AccountDashboardPage';

test('should render the dashboard page', () => {
  const wrapper = shallow(<AccountDashboardPage />);
  expect(wrapper).toMatchSnapshot();
})
