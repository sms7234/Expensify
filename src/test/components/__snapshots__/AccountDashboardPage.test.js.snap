// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the dashboard page 1`] = `
<div>
  <Connect(AccountSummary) />
  <Connect(AccountListFilters) />
  <Connect(AccountList) />
</div>
`;
